test/riemann/codec_test.clj

```diff
@@ -102,3 +102,24 @@
         expected (dissoc original :empty-attr)]
     (is (= (msg {:events [expected]})
            (roundtrip {:events [original]})))))
+
+(defn not-serializable-event
```

Comment:
It looks like this emits both serializable and nonserializable events. Maybe a different function name is in order?